external/fv3fit/fv3fit/emulation/zhao_carr/filters.py

```diff
@@ -0,0 +1,18 @@
+from typing import Set
+from fv3fit.emulation.types import TensorDict
+import dataclasses
+import numpy as np
+
+
+@dataclasses.dataclass
+class HighAntarctic:
+    high_antarctic_only: bool = True
+
+    @property
+    def input_variables(self) -> Set[str]:
+        return {"latitude", "surface_air_pressure"}
+
+    def __call__(self, elm: TensorDict):
+        lat = elm["latitude"] < np.deg2rad(-60)
+        ps = elm["surface_air_pressure"] < 700e2
+        return (ps & lat)[0]
```

Comment:
`[0]` is to squeeze dimensions, I assume.  Maybe add a module level comment  or protocol that returns should be applicable only to a single element if we don't want broadcasts for the unbatched `dataset.filter` operation.

Reply:
Correct. it needs to return a scalar bool. I documented this.